docs/notebooks/03_merge.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>Not sure if it might make sense to have this loop running with <code>process_map</code> if we imagine many datasets?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>only if it's unbearably slow... a for loop is easier to follow along for the reader. </p>